src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -14,8 +14,10 @@
     <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
+    <ToolOS Condition="'$(OS)' == 'Windows_NT'">win7</ToolOS>
+    <ToolOS Condition="'$(OS)' != 'Windows_NT'">ubuntu.14.04</ToolOS>
```

Comment:
Ok, I will remove this then and from coreclr repo I'll do something similar to your PR where you check if corerun exists, and set the NugetRuntimeId over there. Thanks for the feedback! @weshaggard with this in mind, should we add more distros and osx to the tool-runtime\project.json runtimes?